src/command/stopper_windows.go

```diff
@@ -5,9 +5,19 @@ import (
 	"strconv"
 )
 
-func (c *CmdWrapper) Stop(sig int, _parentOnly bool) error {
+func (c *CmdWrapper) Stop(sig int, parentOnly bool) error {
 	//p.command.Process.Kill()
-	kill := exec.Command("TASKKILL", "/T", "/F", "/PID", strconv.Itoa(c.Pid()))
+	log.
+		Debug().
+		Int("pid", c.Pid()).
+		Bool("parentOnly", parentOnly).
+		Msg("Stop Windows process.")
+
+	if parentOnly {
+		kill := exec.Command("C:\Windows\System32\taskkill.exe", "/F", "/PID", strconv.Itoa(c.Pid()))
```

Comment:
Why do we need an absolute path here?

Reply:
SonarQube complained about a relative path https://github.com/F1bonacc1/process-compose/runs/48774929711
but unfortunately after issue is fixed previous report is no longer accessible.
I can put it back so you check the report?